sssd_test_framework/hosts/__init__.py

```diff
@@ -1,2 +1,3 @@
 """SSSD multihost hosts."""
+
```

Comment:
Looks like your IDE is updating this file?

Reply:
I think I first saw failures in the tox CI pipeline but, I was able to see the failures locally too.   I suspect it's related to the black version updates that was mentioned recently.   I added the commit with changes in order to correct the failure thinking that everyone would start seeing that.  I can drop those changes if we are changing the config or pinning black to a specific version as the fix.   

@justin-stephenson were you looking into the issue with black throwing errors recently?

Reply:
https://github.com/SSSD/sssd/pull/7155 was merged to address black changes in sssd tests, but haven't pinned to a specific version yet. Changes to linting requirements are due to https://black.readthedocs.io/en/stable/the_black_code_style/future_style.html

Reply:
@justin-stephenson Ok, so we're seeing black failures in other PRs as well that need something like the commit I have here.  Should I pull that our of this PR and add it to a separate one?

Reply:
A separate PR would be nice if it affects files unrelated to this keycloak PR.

Reply:
ok, I created PR #75 for that an removed the commit from this PR.